### ISSUE TITLE
Csound QT 64bit v6.09.0 doesn't run 

### ISSUE THREAD

**Issue by santibogacz** [csound/csound#822]
Hello!

I use Windows 8.1 Pro. I downloaded Csound's latest version (64bit v6.09.0) and can't run it because the python27.dll is missing. I checked out for some help in the forum. So went and installed  python-2.7.13 32-bit version (as in the forum said that Csound need the 32 bit version to run correctly) and still can't run Csound. Copied the python27.dll file to the bin folder (as it was recommended in the forum) and still can't run the program.

Does anybody know how to solve this problem?

**Comment by gogins** [comment]
The forum is not correct.

Csound in the Windows installer is now 64 bit CPU architecture. If you
install the 64 bit version of Python your problem should go away.

Or, install the latest version of Csound from here (
https://github.com/csound/csound/releases/download/6.09.1/Setup_Csound6_x64_6.09.1.exe)
where you can turn off the Python dependency.

Regards,
Mike


-----------------------------------------------------
Michael Gogins
Irreducible Productions
http://michaelgogins.tumblr.com
Michael dot Gogins at gmail dot com

**Comment by santibogacz** [comment]
Thanks very much for the answer!

Which forum would be the correct one?

Installed the program with that installation and ran perfectly. However, now the "Reading CSD failed...stopping" message started appearing, something that happened to me long ago. I tried to solve it the way I did last time, by moving the csd file to another folder but doesn't seem to work.

Sorry for the disturbance and again, thanks a lot for the help

Yours,
Santiago Bogacz
